app/assets/stylesheets/home.css.scss

```diff
@@ -29,8 +64,9 @@
   }
 }
 
+
 #intro {
-  margin-top: 10vw;
+  margin-top: 3em;
   margin-bottom: 10vw;
```

Comment:
Viewport units are very very young. If you want the best support across browsers stick to: em for vertical spacing and % for horizontal spacing.